migration-guides/v7.0.0.md

```diff
@@ -367,4 +368,17 @@ Prior to v7, passing in a custom class name to the modal would override the defa
   // add additional custom styles
 }
 ```
-This logic applies to both the `className` and `overlayClassName` props, whether the use passes in a string or an object.
\ No newline at end of file
+This logic applies to both the `className` and `overlayClassName` props, whether the use passes in a string or an object.
+
+## 16. `<DateInput />`s dependency on react-datepicker was upgraded from v1 to v4
+This introduces the following breaking changes:
+- [2.0.0](https://github.com/Hacker0x01/react-datepicker/releases/tag/v2.0.0)
+  - `moment` is deprecated by `date-fns`
+- [3.0.0](https://github.com/Hacker0x01/react-datepicker/releases/tag/v3.0.0)
+  - Replace inlineFocusSelectedMonth with focusSelectedMonth
+- [4.0.0](https://github.com/Hacker0x01/react-datepicker/releases/tag/v4.0.0)
+  - Upgraded `react-popper` dependency from v1 to v2
+
+More details: https://github.com/Hacker0x01/react-datepicker/releases.
+
+Of these breaking changes, the deprecation of `moment` has the highest likelihood of affecting your application. This will only impact your application if you referenced the `moment` library elsewhere in the codebase **without** installing the package directly as a dependency.
```

Comment:
Nice. And hopefully, very few people have done this.

Reply:
The reason I knew to include this is because I've done it before 😱 🙊